packages/bpk-component-datatable/src/BpkDataTable.js

```diff
@@ -60,11 +60,16 @@ const sortList = ({ sortBy, sortDirection, list }) => {
 };
 
 class BpkDataTable extends Component {
-  constructor({ rows, children }) {
+  constructor({ rows, children, defaultColumnSortIndex }) {
     super();
 
-    const sortBy = children.length > 0 ? children[0].props.dataKey : undefined;
-    const sortDirection = SortDirection.ASC;
+    const sortBy =
+      children.length > 0
+        ? children[defaultColumnSortIndex].props.dataKey
+        : undefined;
+    const sortDirection =
+      children[defaultColumnSortIndex].props.defaultSortDirection ||
```

Comment:
`defaultSortDirection` should be documented on the `BpkDataTableColumn`

Reply:
Never mind-- this is supported by `react-virtualised`'s `Column`: https://github.com/bvaughn/react-virtualized/blob/master/docs/Column.md